### PR TITLE
Add a headlesswait parameter

### PR DESCRIPTION
**Pull request by valentinedwv**
Headless rendering of one site requires a >1 second way to get an injected jsonld script.

fails in google schema tester.
See it in the local chrome.

This hack lets' the website render and we see the two jsonld blocks.

branch lands after dev_mm_robots... since that has a call to GetSourceByName which is needed to get the headless parameter from a source

### CODE REVIEW

**Comment by nein09** [comment]
@valentinedwv this PR is against master - did you want to do it against dev?

**Comment by fils** [comment]
@valentinedwv agree with Melinda..  could you do this pull request again dev.

**Comment by valentinedwv** [comment]
Changed to dev

File: internal/summoner/acquire/headlessNG.go [hunk 1]
```diff
@@ -231,7 +239,7 @@ func PageRender(v1 *viper.Viper, mc *minio.Client, logger *log.Logger, timeout t
 			});
 		}
 
-		function retry(fn, retriesLeft = 3, interval = 1000) {
+		function retry(fn, retriesLeft = 3, interval = 3000) {
```
Comment (nein09) [inline internal/summoner/acquire/headlessNG.go:242]: If I remember right, that interval was pretty arbitrary. Would it make sense to make it a multiple of HeadlessWait?

File: internal/summoner/acquire/headlessNG.go [hunk 2]
```diff
@@ -45,7 +45,7 @@ func HeadlessNG(v1 *viper.Viper, mc *minio.Client, m map[string][]string, db *bo
 		})
 
 		for i := range m[k] {
-			err := PageRender(v1, mc, logger, 60*time.Second, m[k][i], k, db) // TODO make delay configurable
+			err := PageRender(v1, mc, logger, 180*time.Second, m[k][i], k, db) // TODO make delay configurable
```
Comment (nein09) [inline internal/summoner/acquire/headlessNG.go:48]: If someone puts a really long HeadlessWait in, does that cause havoc here?

**Comment by fils** [comment]
This looks good..  sorry for my delay

**Pull request merged**